src/OpenTelemetry.SemanticConventions/scripts/generate.ps1

```diff
@@ -2,28 +2,29 @@ $SCRIPT_DIR = $PSScriptRoot
 $ROOT_DIR = "${SCRIPT_DIR}/../"
 
 # freeze the spec version to make SemanticAttributes generation reproducible
-$SPEC_VERSION = "1.25.0"
-$GENERATOR_VERSION = "latest"
+$SEMCONV_VERSION = "1.26.0"
```

Comment:
Any reason why you're not using the latest version 1.27.0?

Reply:
No specific reason, I wanted to keep the changes at minimum at this first PR.